### PR TITLE
Remove old amis, update ubuntu 2204 ami

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?

Remove old amis
Update ubuntu owner

### CODE REVIEW

**Review by mcalmer — APPROVED** [review]

File: backend_modules/aws/base/ami.tf [hunk 1]
```diff
@@ -256,7 +236,7 @@ data "aws_ami" "centos7" {
 data "aws_ami" "ubuntu2204" {
   most_recent = true
   name_regex  = "^ubuntu/images/hvm-ssd/ubuntu-jammy-22.04"
-  owners      = ["679593333241"]
+  owners      = ["099720109477"]
```
Comment (rjmateus) [inline backend_modules/aws/base/ami.tf:239]: why did you change the owner for the ubuntu image? If I got it right, this owner is for an image that is not official.
We should keep the official one, otherwise we can be using corrupt images and pay more than needed.

**Review by rjmateus — CHANGES_REQUESTED** [review]
